scripts.js

```diff
@@ -1,7 +1,8 @@
+import 'babel-polyfill';
 import loadImage from "blueimp-load-image";
 import todaysCuriosity from './todays-curiosity';
 import george from './george.jpg';
-
+import getContributorsHTML from './footer';
```

Comment:
@TakeAWalk What you had here earlier wasn't quiet right `import footer from './footer';`
with ES6 modules, you would need to have exported something i.e. footer from footer.js, but in this case were were just running footer.js, not getting a function from it so `import './footer;` would have been fine.

Though I decided to export a function because I didn't like the idea of adding html to the page in some random file, since `script.js` is where all of the listeners and query selectors are set up, I wanted to keep it that way. So `footer.js` keeps or the logic, but we can see what the final result is in a higher abstraction still in this file down the bottom with 
``` javascript
getContributorsHTML().then(
  contributorsHtml => document.querySelector("#footer").innerHTML = contributorsHtml
);
```